test/test_publish.py

```diff
@@ -432,3 +434,27 @@ def test_regression_atom_feed_update(dvcs_type, tmpdir):
         a_content = a.find('{http://www.w3.org/2005/Atom}content')
         b_content = b.find('{http://www.w3.org/2005/Atom}content')
         assert a_content.text != b_content.text
+
+def test_gh1209(tmpdir):
```

Comment:
A docstring or changing the test name to something indicative of what is being tested will prevent confusion in the future. Maybe `test_branch_name_is_also_filename` ?